src/main/app/claims/models/response/partDefenceAdmission.ts

```diff
@@ -0,0 +1,62 @@
+import { Moment } from 'moment'
+import { MomentFactory } from 'shared/momentFactory'
+
+import { ResponseCommon } from 'claims/models/response/responseCommon'
+import { ResponseType } from 'claims/models/response/responseType'
+
+import { PaymentOption } from 'claims/models/response/core/paymentOption'
+import { RepaymentPlan } from 'claims/models/response/core/repaymentPlan'
+
+import { StatementOfMeans } from 'claims/models/response/statement-of-means/statementOfMeans'
+import { DefendantEvidence } from 'response/form/models/defendantEvidence'
+import { DefendantTimeline } from 'response/form/models/defendantTimeline'
+import { YesNoOption } from 'claims/models/response/core/yesNoOption'
+
+export interface PartAdmissionResponse extends ResponseCommon {
+  responseType: ResponseType.PART_ADMISSION
+  isAlreadyPaid: YesNoOption
+  howMuchHaveYouPaid: {
+    amount: number,
+    date: Moment,
+    paymentMethod: string
+  }
+  defence: string,
+  timeline: DefendantTimeline
+  evidence: DefendantEvidence
+  paymentOption?: PaymentOption
+  paymentDate?: Moment
+  repaymentPlan?: RepaymentPlan
+  statementOfMeans?: StatementOfMeans
+}
+
+export namespace PartAdmissionResponse {
```

Comment:
as we have `PartialAdmission` as a draft why don't we have `PartAdmissionResponse`?